theano/misc/pkl_utils.py

```diff
@@ -166,7 +166,7 @@ def write_array(f):
                 zipadd(write_array, self.zip_file, name)
                 self.seen[id(obj)] = 'cuda_ndarray.{0}'.format(name)
             return self.seen[id(obj)]
-        super(PersistentCudaNdarrayID, self).__call__(obj)
+        return super(PersistentCudaNdarrayID, self).__call__(obj)
```

Comment:
Can you add a test for this case?
Le 7 mai 2015 17:00, "dmitriy-serdyuk" <notifications@github.com> a écrit :

> In theano/misc/pkl_utils.py
> <https://github.com/Theano/Theano/pull/2887#discussion_r29895542>:
>
> > @@ -166,7 +166,7 @@ def write_array(f):
> >                  zipadd(write_array, self.zip_file, name)
> >                  self.seen[id(obj)] = 'cuda_ndarray.{0}'.format(name)
> >              return self.seen[id(obj)]
> > -        super(PersistentCudaNdarrayID, self).__call__(obj)
> > +        return super(PersistentCudaNdarrayID, self).__call__(obj)
>
> It used regular pickler in case of Cuda Ndarray instead of custom one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2887/files#r29895542>.
>